### DISCUSSION TITLE
Constrution Timelapse with hour and days

### DISCUSSION THREAD

**Discussion by Filtex1** [gopro/labs#258] [Q&A]
Good
I check the existence of timelapse for long time in which it is possible to create start time and end time. I would like to know if anyone can help me to add that this occurs only on the intended days. I also take the opportunity to ask if anyone has a solution to access the camera through the application when it turns off after taking a photo, or if there is a possibility of doing so in some way.
Thanks

**Comment by dnewman-gpsw** [comment]
When the camera is off, you can't access it.  However if the Wireless controls are still activate, within the first hours of the timelapse only, you can turn the camera on, but you have to cancel the timelapse. So there is no practical way to do you ask.  A potential workaround to monitor the time-lapse, is it try using the upload feature.  https://gopro.github.io/labs/control/dailytl/

**Comment by Videocamguy** [comment]
OK, setting another camera with Battery only no mains.  On this Code what does the  mPdP  (Mode Photo, what does the dP  mean?)

mPdP>05:59<20:00!811SQ~!06:00S!1R

Thanks for any info

Gord
